client/apps/game/src/ui/components/resources/entity-labor-table.tsx

```
@@ -0,0 +1,29 @@
+        signer: dojo.account.account,
+        entity_id: entityId,
+        resource_type: resourceType,
+        amount: producedAmount,
+      });
+    } catch (error) {
+      console.error("Failed to start production", error);
+    }
+  };
+
+  return Object.entries(LaborIds).map(([_, laborId]) => {
+    if (isNaN(Number(laborId))) return null;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)